apps/01PigLatin.js

```diff
@@ -7,11 +7,108 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer:
+  //we take the word argument and convert it is a variable
+    //now our data is clean
+      //and we can make a fair comparison between our word letters and the vowel              //letters
+  word  = word.toLowerCase();
```

Comment:
Almost. The reason for .ToLowerCase() is to "Scrub" the input so that we only need to check for a lowercase a,e,i,o,u or y. Had we not used this, we would need to add 6 more if-statements to check for the uppercase versions as well. 'a' != 'A'